pkg/rpcclient/waiter/waiter.go

```diff
@@ -87,6 +89,18 @@ type Null struct{}
 type PollingBased struct {
 	polling RPCPollingBased
 	version *result.Version
+	config  PollConfig
+}
+
+// PollConfig is a configuration for PollingBased waiter.
+type PollConfig struct {
+	// PollInterval is a time interval between subsequent polls. If not set, then
+	// default value is a half of configured block time (in milliseconds), and if
+	// that value is 0 (which is possible for old RPC servers), then default is 1s.
```

Comment:
Not the case now.

Reply:
Yep, left from the old version of this PR.

Reply:
Fixed.